coprarna_aux/copraRNA2_position_script_for_evo_precalculated_alignments_w_ooi.R

```diff
@@ -4,11 +4,24 @@
 
 # dependency: CopraRNA_available_organisms.txt
 
-# R --slave -f ../copraRNA2_position_script_for_evo_precalculated_alignments_w_ooi_fast3.r --args NC_000913
+# R --slave -f ../copraRNA2_position_script_for_evo_precalculated_alignments_w_ooi_v3.R --args NC_000913
```

Comment:
??? what is this comment about?

Reply:
Do you mean "CopraRNA_available_organisms.txt"? that file should be in the CopraRNA working folder anyway

Reply:
solved by #20